spec/controllers/followings_controller_spec.rb

```diff
@@ -0,0 +1,101 @@
+require 'spec_helper'
+
+describe FollowingsController do
+  describe "GET index" do
+    it_behaves_like "a security guard" do
+      let(:action) { get :index }
+    end
+
+    it "sets @followings to users followed by current user" do
+      bob = Fabricate :user
+      set_current_user bob
+      jane = Fabricate :user
+      alex = Fabricate :user
+      following = Fabricate :following, follower: bob, followed: jane
+      following1 = Fabricate :following, follower: jane, followed: alex
+      get :index
+      expect(assigns(:followings)).to eq([following])
+    end
+  end
+
+  describe "DELETE destroy" do
+    let(:bob) { Fabricate :user }
+    let(:jane) { Fabricate :user }
+    let(:following) { following = Fabricate :following, follower: bob, followed: jane }
+    before { set_current_user bob }
```

Comment:
Be careful with `let` and `before` I find your specs in this group very hard to follow because you have to keep jumping up to this section to read them. I would recommend setting up your specs in each `it` as much as possible.